theano/sandbox/rng_mrg.py

```diff
@@ -804,101 +803,125 @@ def pretty_return(self, node_rstate, new_rstate, sample, size, nstreams):
         node_rstate.default_update = new_rstate
         return sample
 
-    def uniform(self, size, low=0.0, high=1.0, ndim=None, dtype=None,
-                nstreams=None, **kwargs):
-        # TODO : need description for parameter 'size', 'ndim', 'nstreams'
+    def uniform(self, size, low=0.0, high=1.0,
+                ndim=None, dtype=None, nstreams=None, target=None):
```

Comment:
Why the change about target? Now it is different then the other method and I would prefer to not change this. So I would like to know why you changed it.

Reply:
Abergon requested support for the `target` parameter.

Reply:
ok, but he was referencing to adding its support to truncated_normal, not change the interface here. Before you changed this code, it was an example of how to add that support.

Reply:
@nouiz In that case: this used to be a `**kwargs` that could only get the keyword argument `'target'` and was not passed to any other function. I considered that, in this case, it would be cleaner to make `target` a direct keyword argument.

Should I revert this change?

Reply:
It is better to have all method implemented the same. Also, the fact of passing `target=None` or not passing target don't do exactly the same thing. This is why it was done like this. We should not change the behavior here.